### PR TITLE
added option in config to check temperature an power or not

### PR DESCRIPTION
**Pull request by afswarm**

### CODE REVIEW

File: fpga.chart.py [hunk 1]
```diff
@@ -62,7 +62,12 @@ def __init__(self, configuration=None, name=None):
         self.xilinx_cmd = self.configuration.get('xilinx_cmd')
         self.fpga_count = 1
         self.dsn = self.configuration.get('dsn')
-        self.metrics = [ 'bytes', 'jobs', 'max', 'temps', 'powers' ]
+        self.check_temp_power = self.configuration.get('check_temp_power')
+
+        if self.check_temp_power is True:
```
Comment (sdressler) [inline fpga.chart.py]: Proposal to reduce code:

    self.metrics = [ 'bytes', 'jobs', 'max' ]
    if self.check_temp_power:
        self.metrics.extend(['temps', 'powers'])

File: fpga.chart.py [hunk 2]
```diff
@@ -187,7 +192,8 @@ def get_data(self):
 
         try:
             data = copy.deepcopy(self.default_data)
-            self.set_fpga_os_status(data)
+            if self.check_temp_power is True:
```
Comment (sdressler) [inline fpga.chart.py]: You can omit the `is True`

File: fpga.conf [hunk 3]
```diff
@@ -1,7 +1,12 @@
-update_every: 1
+update_every: 2
```
Comment (sdressler) [inline fpga.conf]: Suggestion for defaults: 1 sec update and temp/power collection disabled.

File: fpga.chart.py (hunk 2 above)
Comment (afswarm) [reply fpga.chart.py]: I think because we are reading it from a file it might be better to be explicit.

Comment (sdressler) [reply fpga.chart.py]: Fine by me, was just a thought wrt Python best practices.

Comment (afswarm) [reply fpga.chart.py]: ok

File: fpga.chart.py [hunk 4]
```diff
@@ -63,11 +63,10 @@ def __init__(self, configuration=None, name=None):
...
         self.dsn = self.configuration.get('dsn')
         self.check_temp_power = self.configuration.get('check_temp_power')
+        self.metrics = [ 'bytes', 'jobs', 'max' ]
 
-        if self.check_temp_power is True:
-            self.metrics = [ 'bytes', 'jobs', 'max', 'temps', 'powers' ]
-        else:
-            self.metrics = [ 'bytes', 'jobs', 'max' ]
+        if self.check_temp_power:
+            self.metrics.extend([ 'bytes', 'jobs', 'max', 'temps', 'powers' ])
```
Comment (sdressler) [inline fpga.chart.py]: Remove 'bytes', 'jobs', 'max', otherwise you have duplicates.

**Review by sdressler — CHANGES_REQUESTED** [review]
Small bug in there...

**Review by sdressler — APPROVED** [review]

**Pull request merged**